Binary files ndoutils-1.4b7.original/src/dbhandlers-3x.o and ndoutils-1.4b7/src/dbhandlers-3x.o differ
diff -ur ndoutils-1.4b7.original/src/dbhandlers.c ndoutils-1.4b7/src/dbhandlers.c
--- ndoutils-1.4b7.original/src/dbhandlers.c	2012-06-12 14:53:04.000000000 +0000
+++ ndoutils-1.4b7/src/dbhandlers.c	2012-06-12 15:10:53.000000000 +0000
@@ -96,7 +96,7 @@
 			buf2=NULL;
 	        }
 	
-	if(asprintf(&buf,"SELECT * FROM %s WHERE instance_id='%lu' AND objecttype_id='%d' AND %s AND %s"
+	if(asprintf(&buf,"SELECT object_id FROM %s WHERE instance_id='%lu' AND objecttype_id='%d' AND %s AND %s"
 		    ,ndo2db_db_tablenames[NDO2DB_DBTABLE_OBJECTS]
 		    ,idi->dbinfo.instance_id
 		    ,object_type
Binary files ndoutils-1.4b7.original/src/ndo2db-3x and ndoutils-1.4b7/src/ndo2db-3x differ
